tapo@smiklosovic/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "description": "Control Tapo Appliances",
+    "uuid": "tapo@smiklosovic",
+    "name": "Tapo Control Panel",
+    "version": "1.0.0",
+    "max-instances": 1,
+    "author": "Stefan Miklosovic",
+    "last-edited": 1722168753
```

Comment:
```suggestion
```

This will fail validation. Run `./validate-spice tapo@smiklosovic` locally. You can also use `./cinnamon-spices-makepot tapo@smiklosovic` when validation succeeds to add translation files.

Reply:
@rcalixte do you know how do I fix that?

Reply:
ah I see, give me a moment.

Reply:
The errors should hopefully be verbose enough but you can expedite things by looking at the file structure from existing applets and mirroring those.